backend/src/main/resources/schema.sql

```diff
@@ -1,5 +1,22 @@
-create table if not exists MEMBER
+create table if not exists MEMBER (
     id bigint auto_increment not null,
     name varchar(255) not null unique
+);
 
-);
\ No newline at end of file
+create table if not exists CONFERENCE_TIME (
+    id bigint auto_increment not null,
+    time int(24) not null unique
+);
+
+create table if not exists ATTENDANCE (
+    id bigint auto_increment not null,
+    day TIMESTAMP DEFAULT FORMATDATETIME(NOW(), 'yyyy-MM-dd'),
+    member_id bigint not null,
+    time_id bigint not null,
+    primary key(id),
+    foreign key(member_id) references MEMBER(id),
+    foreign key(time_id) references CONFERENCE_TIME(id)
+);
```

Comment:
```suggestion
create table if not exists ATTENDANCE (
    id bigint auto_increment not null,
    day TIMESTAMP DEFAULT FORMATDATETIME(NOW(), 'yyyy-MM-dd'),
    member_id bigint not null,
    time_id bigint not null,
    
    primary key(id),
    
    foreign key(member_id) references MEMBER(id),
    foreign key(time_id) references CONFERENCE_TIME(id)
);
```

ddl도 결국 sql 언어이기 때문에 저의 경우 가독성을 위해서 이런식으로 정리하는 편이랍니다~